modules/auth/webauthn/webauthn.go

```diff
@@ -68,7 +68,7 @@ func (u *User) WebAuthnIcon() string {
 
 // WebAuthnCredentials implementns the webauthn.User interface
 func (u *User) WebAuthnCredentials() []webauthn.Credential {
-	dbCreds, err := auth.GetWebAuthnCredentialsByUID(u.ID)
+	dbCreds, err := auth.GetWebAuthnCredentialsByUID(db.DefaultContext, u.ID)
```

Comment:
Pity that we can't fix that at the moment as it is used in the templates.

Reply:
We have `ctx` in templates (#26254).

Reply:
Yes, but not yet for any usecase, correct?
Or can it simply be used in every template even those that don't have it?

Reply:
It's safe to use `ctx` in any template. I was going to refactor more, but there is an open PR #26745 which would conflict with the `ctx` refactoring. I do not want to introduce too many conflicts.

Reply:
I could not find a template that is using this function

Reply:
There are a lot.

![image](https://github.com/go-gitea/gitea/assets/2114189/83eedfa7-09a3-4e82-8bce-257cd70f7113)

Reply:
I mean `GetWebAuthnCredentialsByUID`

Reply:
You are right. It think it's safe to remove `WebAuthnCredentials`

(While as a refactoring PR, search&replace is good enough, I don't consider the removal is a must, the removal could be done later)